lib/template/config/initializers/timeout.rb

```diff
@@ -0,0 +1,3 @@
+if Config.timeout > 0
```

Comment:
Should we call this file `rack_timeout.rb` maybe? We recently introduced database timeouts as well, so it might help disambiguate.